x/posts/internal/types/post.go

```diff
@@ -9,18 +9,18 @@ import (
 )
 
 // ---------------
-// --- Post id
+// --- Post newID
```

Comment:
I think this is a refactoring error

```suggestion
// --- Post id
```